src/list-sorting/index.js

```diff
@@ -1,3 +1,31 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  // check if the haystack is an one dimensional array or multi
```

Comment:
What happened? Formatting is worse than in the other file. You start to use `var` for variable declaration. Makes me wonder if you wrote the other one yourself.

Reply:
Yes...I solved them all by myself...I use the 3 though...but will change up to const and let variable declaration fully...
Please...I know there will be a smarter way to do this...can u give one or two hint...I wil implement it myself..nd send to u...thanks Bond....